frontend/src/App.tsx

```diff
@@ -72,7 +73,8 @@ const App = (): ReactElement => {
             <Route exact path="/policies" component={Policies} />
             <Route path="/landlord/:landlordId" component={LandlordPage} />
             <Route path="/apartment/:aptId" component={ApartmentPage} />
-            <Route component={NotFoundPage} />
+            <Route exact path="/notfound" component={NotFoundPage} />
```

Comment:
`<Switch />` in the newest version of `react-router-dom` is now deprecated, so you'll want to replace it with `<Routes />` which is documented on [this stack overflow post](https://stackoverflow.com/a/69849271). There are also some other syntax changes you'll have to make that the post describes ^^